src/object.d

```diff
@@ -3190,11 +3199,24 @@ void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
  * Create RTInfo for type T
  */
 
+template RTInfoImpl(size_t[] pointers)
+{
+    immutable size_t[pointers.length] data = pointers[];
+    immutable RTInfoImpl = data.ptr;
```

Comment:
> Should we at least allocate it lazily, there are typically big regions of NO_SCAN data, e.g. strings and number arrays.

I don't think lazy allocation of `is_pointer` helps much, as it is allocated per pool. It's very unlikely that a full pool doesn't contain any pointer. What might help is to have different pools (and free-lists) for NO_SCAN data and data with pointers. That doesn't allow migrating between the pools via `gc_setAttr` or `gc_emplace`, though.